lib/ProductOpener/ContributionKnowledgePanels.pm

```diff
@@ -0,0 +1,158 @@
+# This file is part of Product Opener.
+#
+# Product Opener
+# Copyright (C) 2011-2023 Association Open Food Facts
+# Contact: contact@openfoodfacts.org
+# Address: 21 rue des Iles, 94100 Saint-Maur des Fossés, France
+#
+# Product Opener is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+=head1 NAME
+
+ProductOpener::ContributionKnowledgePanels - Generate knowledge panels around contribution
+
+=head1 SYNOPSIS
+
+This is a subpart of KnowledgPanels where we concentrate around contribution informations
+
+=cut
+
+package ProductOpener::ContributionKnowledgePanels;
```

Comment:
I would prefer to reverse the name: KnowledgePanelsContribution, so that it's easier to see related modules (as we have for API modules that all start with API)

Reply:
Indeed I have the same problem with APITest.pm that I would prefer named TestAPI.pm :-)

I'll do it.